### PR TITLE
docs: Add stale.yml file for stalebot

### PR DESCRIPTION
**Pull request by ekowtaylor**
Signed-off-by: Ekow Taylor <ekowtaylor@gmail.com>

## Summary
This implements the config file for a GitHub App built with Probot that closes abandoned Issues and Pull Requests after a period of inactivity.

## Test Plan
Privately on another repo

### CODE REVIEW

**Comment by ekowtaylor** [comment]
@electronjoe , @themarwhal please stamp this when you have sometime. The app has already been installed and this is the last part left to enable it.

**Comment by electronjoe** [comment]
Ekow can you fix the linting warnings?

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Number of days of inactivity before an issue becomes stale
```
Comment (electronjoe) [inline .github/stale.yml:2]: Prefix entire file with --- per YAML lint warning

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+daysUntilStale: 45
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - status: status: willfix
```
Comment (electronjoe) [inline .github/stale.yml]: YAML linting doesn't like this line - are we able to quote `status: wilfix`? or do we need to disable YAML lint for this line in the file (done elsewhere in other workflow yaml files if you look around).

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# Label to use when marking an issue as stale
+staleLabel: status: wontfix
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This pull request has been automatically marked as stale because it has not
```
Comment (electronjoe) [inline .github/stale.yml]: Here and elsewhere. YAML lint findings, drop trailing whitespace please!

**Review by electronjoe — CHANGES_REQUESTED** [review]

File: .github/stale.yml (hunk 2 above)
Comment (ekowtaylor) [reply .github/stale.yml]: I'll work around this

File: .github/stale.yml (hunk 3 above)
Comment (ekowtaylor) [reply .github/stale.yml]: Will work around them all

**Comment by ekowtaylor** [comment]
@electronjoe I just submitted a new commit which should address the issues

**Review by electronjoe — APPROVED** [review]

**Pull request merged**